db/migrate/20231121120031_create_system_notifications.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSystemNotifications < ActiveRecord::Migration[7.0]
+  def change
+    create_table :system_notifications do |t|
```

Comment:
Please add missing colukn of state

Would want to avoid using boolean column of `read` and instead manage a `state` field